hypervisor/debug/profiling.c

```diff
@@ -871,7 +871,7 @@ int32_t profiling_vm_list_info(struct acrn_vm *vm, uint64_t addr)
 	vm_info_list.vm_list[vm_idx].num_vcpus = i;
 	vm_info_list.num_vms = 1;
 
-	for (j = 0U; j < CONFIG_MAX_VM_NUM; j++) {
+	for (j = 0U; j < CONFIG_MAX_VM_NUM-1; j++) {
 		tmp_vm = get_vm_from_vmid(j);
```

Comment:
why we use "j" here? should be vm_idx?

Reply:
yes, makes sense to use vm_idx, will fix it